python/ray/_private/internal_api.py

```diff
@@ -30,20 +38,16 @@ def memory_summary(
 ):
     from ray.dashboard.memory_utils import memory_summary
 
-    address = services.canonicalize_bootstrap_address_or_die(address)
+    get_state_from_address(address)
```

Comment:
is this the intended use for `state`?
```suggestion
    state = get_state_from_address(address)
```